packages/travix-ui-kit/tests/unit/stepper/stepper.setValue.spec.js

```diff
@@ -0,0 +1,78 @@
+import Stepper from '../../../components/stepper/stepper';
+
+describe('Stepper: setValue', () => {
```

Comment:
We should define if it will be a controlled or uncontrolled input, but that's no reason to change the value using the component instance.